packages/suite/src/middlewares/wallet/storageMiddleware.ts

```diff
@@ -19,6 +18,7 @@ import { WalletAction } from '@wallet-types';
 import { getDiscovery } from '@wallet-actions/discoveryActions';
 import { isDeviceRemembered } from '@suite-utils/device';
 import { serializeDiscovery } from '@suite-utils/storage';
+import { Trade } from '@suite-actions/storageActions';
```

Comment:
ambiguous names/types `Trade` in storageActions and coinmarketReducer.
maybe both should be placed in shareable file `@wallet-types/coinmarketCommonTypes` ?

Reply:
done